src/Sylius/Bundle/AddressingBundle/Form/EventListener/BuildAddressFormSubscriber.php

```diff
@@ -137,6 +137,8 @@ private function createProvinceCodeChoiceForm(CountryInterface $country, $provin
         return $this->formFactory->createNamed('provinceCode', ProvinceCodeChoiceType::class, $provinceCode, [
             'country' => $country,
             'auto_initialize' => false,
+            'label' => 'sylius.form.address.province',
```

Comment:
@silverbackdan I guess the issue you encountered is fixed by adding label for `provinceName` field created in `createProvinceNameTextForm` (which is also done in this PR). Can you check if it is still fixed without this changes here?